Kudu.Core/Settings/EnvironmentSettingNames.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kudu.Core.Settings
+{
+    class EnvironmentSettingNames
+    {
+        public const string ContainerName = "CONTAINER_NAME";
```

Comment:
Would these environment variables be passed to Kudu container when it is launched in SeaBreeze ? We don't pass them to Kudu for Linux platform currently.

Reply:
@sanchitmehta Yea, the CONTAINER_NAME environment variable is passed in when the container starts.